docs/users.md

```diff
@@ -2,12 +2,76 @@
 
 HTTP Verb | Endpoint                   | Description                | Link
 ----------|----------------------------|----------------------------|---------------------------
+GET       | `/users/find?google_id={google_id}`         | Retrieve a user by their Google ID.         | [Link](#find-user)
```

Comment:
I'd say we leave the query parameters out of this endpoint in the top level table

Reply:
Updated accordingly